.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -5,27 +5,58 @@ labels: bug, sg-library
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+### Describe the bug
+
+*A clear and concise description of what the bug is.*
+
+### To Reproduce
 
-**To Reproduce**
 Steps to reproduce the behavior:
-1. Go to '...'
-2. Execute '...'
-4. See error
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+1. *Go to '...'*
+2. *Execute '...'*
+3. *See error*
+
+### Observed behavior
+
+*Describe or paste the exact output that demonstrates the bug (for example, the stack trace if it is an exception).*
+
+### Expected behavior
+
+*A clear and concise description of what you expected to happen.*
+
+### Environment
+
+Operating system: *for example: macOS, Ubuntu*
+
+Python version: *for example: 3.7.2*
+
+Package versions: *for example: stellargraph==0.8.3, tensorflow==2.0.0*
+
+*The following Python code can automatically compute the data for this section (for example: on macOS, copy the code and run `pbpaste | python -`):*
+
+~~~python
+import subprocess, platform, sys
+packages=subprocess.run(["pip", "freeze"], stdout=subprocess.PIPE, stderr=subprocess.STDOUT).stdout.decode("utf-8")
+print("""Operating system: `{}`
+Python version:
+```
```

Comment:
I think that the multiline string constant here could be compressed substantially into fewer lines to make this more compact?

Reply:
also should we consider something that will run on python2 (just-in-case?!)

Reply:
> I think that the multiline string constant here could be compressed substantially into fewer lines to make this more compact?

I considered this, but I thought that users may appreciate being able to understand the code more easily, since we're asking them to run code on their machine... however, now that I think about it a user is presumably already running our code (in the form of the library itself), so it's probably fine. I've switched to a more compact form, and added a suggestion that it can be run in a jupyter notebook too.

> also should we consider something that will run on python2 (just-in-case?!)

Good idea, I've switched to using `Popen` in a way that seems to work on Python 2 and 3.

Reply:
I had second thoughts about the python2 support; would need an additional from __future__ import print_function

Reply:
I think it's ok, because it's a single argument the parens are just grouping (i.e. `(x)` is the same as `x`).